src/app/pages/OnDemandTvPage/OnDemandTvPage.tsx

```diff
@@ -152,24 +154,27 @@ const OnDemandTvPage = ({ pageData, mediaIsAvailable, MediaError }) => {
             : []
         }
       />
-      <StyledGelPageGrid
+      {/* @ts-expect-error: Legacy grid expects `children` to be passed as props. However, due to coding best practices, we must nest children between the opening and closing tags */}
+      <GelPageGrid
```

Comment:
Not necessarily for now, but it would be good to drop the GelGrid stuff and just use pure CSS Grid since we don't actively support IE anymore.

We've done this in the ArticlePage and MediaArticlePage components for reference, but may be a follow up task to this PR.

Reply:
Thanks, yes, this is the bare minimum to get the page to TS, so that we can take advantage of it when we add the MediaLoader. I will look into this grid stuff if I get the chance, but in a separate PR 😄